packages/@angular/cli/commands/init.run.ts

```diff
@@ -1,12 +1,16 @@
 import * as chalk from 'chalk';
+import {checkYarnOrCNPM} from '../utilities/check-package-manager';
+import {CliConfig} from '../models/config';
 import LinkCli from '../tasks/link-cli';
 import NpmInstall from '../tasks/npm-install';
 import { validateProjectName } from '../utilities/validate-project-name';
 
+
 const Promise = require('../ember-cli/lib/ext/promise');
 const SilentError = require('silent-error');
 const normalizeBlueprint = require('../ember-cli/lib/utilities/normalize-blueprint-option');
 const GitInit = require('../tasks/git-init');
+const config = CliConfig.fromProject();
```

Comment:
The project's config  will never exist in `new`. You probably want `fromGlobal()`?